cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -314,10 +314,10 @@ func TestEvaluateExecute(t *testing.T) {
 					validateREADME(t, data)
 				},
 				filepath.Join("result-directory", string(evaluatetask.IdentifierWriteTests), "symflower_symbolic-execution", "golang", "golang", "plain", "evaluation.log"): func(t *testing.T, filePath, data string) {
-					assert.Contains(t, data, "coverage objects: [{")
+					assert.Contains(t, data, "\"msg\":\"coverage objects\",\"objects\":\"[{")
```

Comment:
commit subject: "Adapting test cases to changes in logging"
And a message body that makes sense to me, please :wink: